semgrep/semgrep/test.py

```diff
@@ -307,7 +306,7 @@ def main(
 def test_main(args: argparse.Namespace) -> None:
     _test_compute_confusion_matrix()
     if len(args.target) != 1:
-        print_error_exit("only one target directory allowed for tests")
+        raise Exception("only one target directory allowed for tests")
```

Comment:
It doesn't really make sense to raise `SemgrepError` here since we're in the tests.